src/main/groovy/grails/plugins/elasticsearch/ElasticSearchContextHolder.groovy

```diff
@@ -45,7 +46,7 @@ class ElasticSearchContextHolder {
      * @return
      */
     SearchableClassMapping getMappingContext(DomainEntity domainClass) {
```

Comment:
Though, I have not fully verified but it seems like this change will affect other functionalities too. Are you sure this is required?